### PR TITLE
Policy Simulation command [WIP]

### PR DESCRIPTION
**Pull request by s1ntaxe770r**
WIP implementation of #79

### CODE REVIEW

**Comment by nyrahul** [comment]
> @nyrahul this is far from done here but what do you think of this current implementation? Also, what does `kubearmor.cfg` look like ?

### kubearmor.cfg
```
defaultPosture: audit # possible values allow, block, audit
```
This is the only configuration that needs to be used as of now since this is required for policy simulation

**Comment by s1ntaxe770r** [comment]
Thanks, also I see over [here](https://github.com/kubearmor/KubeArmor/pull/727) that default posture is passed as an environment variable, Should the command take this into account? If yes should it take preference if a `kubearmor.cfg` is passed in

File: simulate/simulate.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+		response.Action = karmorPolicy.Spec.Process.Action
+		response.Result = "Permission Denied"
+		PrintResults(response, "Block")
+		return nil
+	} else if len(karmorPolicy.Spec.File.MatchPaths) > 0 {
+		response.Resource = karmorPolicy.Spec.File.MatchPaths[0].Path
+		response.Policy = o.Policy
+		response.Severity = 1
+		response.Source = karmorPolicy.Spec.File.MatchPaths[0].Path
+		response.Data = "syscall=SYS_FOPEN"
```
Comment (s1ntaxe770r) [inline simulate/simulate.go:78]: hey, @nyrahul  is this the right syscall?

Comment (nyrahul) [reply simulate/simulate.go:78]: Nope, you can setup kubearmor and check the telemetry events for file open.

**Comment by s1ntaxe770r** [comment]
@nyrahul here's how it's looking right now, still need to handle more conditions but i hope i am on the right track here
image

**Comment by Ankurk99** [comment]
Hey @s1ntaxe770r are you working on this issue?

**Comment by s1ntaxe770r** [comment]
Right now no , I got stuck on some on parts of the design spec so I'm working on other issues right now

**Pull request closed without merging**